test/e2e/set/scyllacluster/scyllacluster_scaling.go

```diff
@@ -94,5 +110,15 @@ var _ = g.Describe("ScyllaCluster", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
+
+		framework.By("Verifying data consistency")
```

Comment:
would it be more concise (and wide-spread) if we make `verifyScyllaCluster` take an optional arg?

Reply:
verifyScyllaCluster now takes an optional func as a parameter